doc/user/cluster_tls.md

```diff
@@ -85,3 +85,27 @@ $ kubectl create secret generic operator-etcd-client-tls --from-file=etcd-ca-crt
 ```
 
 Pass `operator-etcd-client-tls` to `operatorSecret` field.
+
+### Access a secure etcd cluster
+
+Assume a secure etcd cluster `example` is up and running.
+
+To access the cluster, use the FQDN `example-client.default.svc.cluster.local`, which matches the SAN of its certificates.
+To add more DNS name or IP, that should be added when generating etcd server's client certs.
```

Comment:
We have links at the top of this guide.

> To use customized DNS name or IP

That's not correct. It should be additional. We have requirements on what domains need to be included.